tests/test-pbr.cpp

```diff
@@ -171,37 +173,41 @@ namespace libsemigroups {
   }
 
   LIBSEMIGROUPS_TEST_CASE("PBR", "006", "exceptions", "[quick][pbr]") {
-    REQUIRE_THROWS_AS(PBR::make({{1}, {4}, {3}, {10}, {0, 2}, {0, 3, 4, 5}}),
-                      LibsemigroupsException);
-    REQUIRE_THROWS_AS(PBR::make({{4}, {3}, {0}, {0, 2}, {0, 3, 4, 5}}),
+    REQUIRE_THROWS_AS(
+        to_pbr_no_checks({{1}, {4}, {3}, {10}, {0, 2}, {0, 3, 4, 5}}),
+        LibsemigroupsException);
+    REQUIRE_THROWS_AS(to_pbr_no_checks({{4}, {3}, {0}, {0, 2}, {0, 3, 4, 5}}),
                       LibsemigroupsException);
 
     REQUIRE_NOTHROW(PBR({{-3, -1}, {-3, -2, -1, 1, 2, 3}, {-3, -2, -1, 1, 3}},
                         {{-3, -1, 1, 2, 3}, {-3, 1, 3}, {-3, -2, -1, 2, 3}}));
+    REQUIRE_NOTHROW(
+        to_pbr_no_checks({{-3, -1}, {-3, -2, -1, 1, 2, 3}, {-3, -2, -1, 1, 3}},
+                         {{-3, -1, 1, 2, 3}, {-3, 1, 3}, {-3, -2, -1, 2, 3}}));
 
     REQUIRE_NOTHROW(PBR({{}, {}}));
 
     REQUIRE_THROWS_AS(
-        PBR::make({{-4, -1}, {-3, -2, -1, 1, 2, 3}, {-3, -2, -1, 1, 3}},
-                  {{-3, -1, 1, 2, 3}, {-3, 1, 3}, {-3, -2, -1, 2, 3}}),
+        to_pbr_no_checks({{-4, -1}, {-3, -2, -1, 1, 2, 3}, {-3, -2, -1, 1, 3}},
```

Comment:
Why is this called no checks but appears to throw here?

Reply:
There are no checks on the arguments before the `PBR` is constructed, so it's possible for bad things to happen before it can be validated. That said, I didn't thoroughly look into what happens during construction, so if nothing bad can happen at that stage, then the `_no_checks` should probably be dropped.